pages/index.js

```diff
@@ -55,7 +54,7 @@ export default function Index() {
               </section>
             </div>
             <p>
-              Peer-to-peer networking offers a new design space for bazaar-style development, as it provides new communication pathways for developers that aren't limited by current processes and workflows. That's why we've decided to pursue a <em>multi-upstream</em> model for Radicle. We've realized by exploring multi-upstream peer-to-peer development makes it far easier for developers and maintainers to develop not just a shared, but a <em>trusted</em> representation grounded in actual source code <em>and</em> secure peer identities.
+              From GitHub to the Linux Kernel Mailing List, we've seen free and open source collaboration take shape as varying workflows & practices. With peer-to-peer technology, we have the ability to uncover new means of collaboration for developers that aren't limited by current user experiences or platforms. Radicle returns control back to the individual and empowers developers to renvision what shared, open, and trusted code collaboration can be.
```

Comment:
It's good but too salesy. Should be more factual I'd say.

Reply:
Okay, I guess that means just reworking the last sentence right? Like saying something along the lines of - "Radicle's multi-upstream model returns control back to thee user... etc.."